ern-core/src/android.ts

```diff
@@ -12,14 +12,14 @@ import semver from 'semver';
 // Default value for android build config
 // ==============================================================================
 
-export const DEFAULT_ANDROID_GRADLE_PLUGIN_VERSION = '4.2.2';
+export const DEFAULT_ANDROID_GRADLE_PLUGIN_VERSION = '7.0.1';
```

Comment:
Specific reason for `7.0.1`? Looks like RN 0.68 ships with `7.0.4`.

Reply:
from changelog 68.0 here as well